### PR TITLE
Glasses Resprite & Inhands

### PR DESCRIPTION
**Pull request by alsoandanswer**
yeah. what it says on the tin, adds inhands, like all my previous inhand PRs

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by NonQueueingMatt** [comment]
What are all the placeholders in the icon files, exactly?

**Comment by alsoandanswer** [comment]
Space fillers, so I can quickly compare two files between each other.
I can remove them if you want.
Wt3LEsLVCw

**Comment by Geevies** [comment]
With `icons/mob/eyes.dmi` deleted, does the on-mob sprites still show up? Also yeah, I think deleting the placeholders is for the best.

**Comment by NonQueueingMatt** [comment]
Delete the placeholders.

**Comment by alsoandanswer** [comment]
@Geevies `icons/mob/eyes.dmi` wasn't deleted, so I don't know what you're talking about

@NonQueueingMatt removed placeholders

File: html/changelogs/wezzy_glasses_resprite.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - imageadd: "Adds new sprites and inhands for mesons. They should be more representative of what they actually do. (Try turning on mesons on and off in your hand and see the magic!)"
```
Comment (NonQueueingMatt) [inline html/changelogs/wezzy_glasses_resprite.yml]: Changelog doesn't actually address everything that's been changed.

Comment (alsoandanswer) [reply html/changelogs/wezzy_glasses_resprite.yml]: Alright I'll fix that

**Pull request merged**